magda-web-client/src/Components/Dataset/Add/SpatialAreaInput/StateSelect.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { FunctionComponent } from "react";
+import ReactSelect from "react-select/async";
+import { ValueType } from "react-select/src";
+import fetch from "isomorphic-fetch";
```

Comment:
Do you need fetch for web-client?

Reply:
I guess it's for compatible with old browser? 
Although itself is not a polyfill, it includes the polyfill.
All other parts of the frontend code use this, so I guess for consistency, better use this?
Plus, it will be SSR ready code.
Your thoughts?

Reply:
🤷‍♂ if it's already being pulled into frontend code I guess it's ok :)